egs/wsj/s5/local/nnet3/run_blstm.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
```

Comment:
Yes agree. I will remove it eventually after I got the result of a good configuration of blstm.

Thanks,
Yiming